ports/libsimpleservo/api/Cargo.toml

```diff
@@ -31,6 +31,7 @@ libloading = "0.5"
 [build-dependencies]
 gl_generator = "0.14"
 serde_json = { workspace = true }
+vergen = "3"
```

Comment:
Any reason we are not using the latest version on crates.io, which is 8.2?

Reply:
Huh. Google Search took me to a very old version. I'm using the latest version now.